File: codewars/python/kyu_8/remove_first_and_last_character.py
"""
Remove First and Last Character

https://www.codewars.com/kata/56bc28ad5bdaeb48760009b0/train/python
"""

def remove_char(s):
    result = ["" for _ in range (len(s) - 2)]
    
    for i in range (1, len(s)-1):
        result[i-1] = s[i]
    
    return "".join(result)

File: codewars/javascript/kyu_8/remove_first_and_last_character.js
"""
Remove First and Last Character

https://www.codewars.com/kata/56bc28ad5bdaeb48760009b0/train/javascript
"""

function removeChar(str){
  let result = [];
  
  for (let i = 1; i < str.length-1; i++){
    result.push(str[i]);
  }
  
  return result.join('');
};
